alpha/apps/kaltura/lib/kParentChildEntryUtils.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * kParentChildUtils is all utils needed for entry parent-child use cases.
+ * @package Core
+ * @subpackage utils
+ *
+ */
+class kParentChildEntryUtils
+{
+	/**
+	 * @param $entryId
+	 * @return string
+	 */
+	public static function getCaptionAssetEntryId($entryId)
+	{
+		$entry = entryPeer::retrieveByPK($entryId);
+		if ($entry && $entry->getParentEntryId())
+		{
+			return $entry->getParentEntryId();
+		}
+
+		return $entryId;
+	}
+
+	/**
+	 * @param $entryIds
+	 * @return array of entryIds - parent entries ids will return for each entry or the entryId itself in case he is not a child entry
+	 * @throws PropelException
+	 */
+	public static function getParentEntryIds($entryIds)
+	{
+		myDbHelper::$use_alternative_con = myDbHelper::DB_HELPER_CONN_PROPEL2;
```

Comment:
Since this util can be called from everywhere, we must not change the course of DB selection here - please remove.